getputremovetest/src/main/java/org/infinispan/json/JsonSetup.java

```diff
@@ -0,0 +1,45 @@
+package org.infinispan.json;
+
+import java.util.UUID;
+
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+
+@State(Scope.Thread)
+public class JsonSetup {
+
+   @Param({"1", "512", "1024"})
+   int stringSize;
+
+   @Param({"new", "main"})
+   String type;
+
+   String toEscape;
+
+
+   @Setup
+   public void setup() {
+      int limit = stringSize * 1024;
```

Comment:
I am thinking it might be better to instead using the stringSize directly, so that a user can override the value as necessary and have more control. Then just change the defaults above to multiplied by 1024.